crates/ruff_linter/src/rules/ruff/rules/sequence_sorting.rs

```diff
@@ -895,6 +895,24 @@ fn multiline_string_sequence_postlude<'a>(
     };
     let postlude = locator.slice(TextRange::new(postlude_start, dunder_all_range_end));
 
+    // If the postlude consists solely of a closing parenthesis
+    // (not preceded by any whitespace/newlines),
+    // plus possibly a single trailing comma prior to the parenthesis,
+    // fixup the postlude so that the parenthesis appears on its own line,
+    // and so that the final item has a trailing comma.
+    // This produces formatting more similar
+    // to that which the formatter would produce.
+    let parenthesis_re = regex::Regex::new(r"^,?([\])}])$").unwrap();
```

Comment:
Is it possible to write this via string find / filter checks, rather than using a regex? Regex tends to be overkill and slower for simple operations like this.

Reply:
Yes, definitely doable -- just _slightly_ less elegant here :)

I'll make the change!

Reply:
As a tip: if we _did_ use a regex, we should ensure that it's only compiled once (like in Python). It can make a _huge_ difference. You can grep for `Lazy<Regex>` as an example of that pattern.

Reply:
Got rid of the regex in 6fa2c43c8629094ae4b918cbb97b94bee432e4ac!